packages/connect-thegraph-template/subgraph/.gitignore

```diff
@@ -59,3 +59,6 @@ typings/
 
 # next.js build output
 .next
+
+# TheGraph artifacts
+generated
```

Comment:
Maybe we should include the `build` folder to the ignored files?

Reply:
Yes! Thank you.